Description: Launch the _gui.tcl directly (instead of launching wish
 outside of it), avoids a lintian warning.
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2011-07-24
--- a/hplj1020.desktop
+++ b/hplj1020.desktop
@@ -2,7 +2,7 @@
 Name=Reactivate HP LaserJet 1018/1020 after reloading paper
 Comment=Click the button in this program when your printer ran out of paper and you have refilled the paper
 # zExec=usb_printerid /dev/usb/lp0
-Exec=wish /usr/share/foo2zjs/hplj10xx_gui.tcl
+Exec=/usr/share/foo2zjs/hplj10xx_gui.tcl
 Terminal=false
 Type=Application
 Icon=/usr/share/pixmaps/hplj1020_icon.png
